mmhuman3d/core/visualization/visualize_smpl.py

```diff
@@ -247,32 +247,30 @@ def _prepare_mesh(poses, betas, transl, verts, start, end, body_model):
                 raise KeyError(
                     f'{str(poses.keys())}, Please make sure that your '
                     f'input dict has all of {", ".join(body_pose_keys)}')
-            num_frame = poses['body_pose'].shape[0]
+            num_frames = poses['body_pose'].shape[0]
             _, num_person, _ = poses['body_pose'].view(
-                num_frame, -1, NUM_BODY_JOINTS * 3).shape
+                num_frames, -1, NUM_BODY_JOINTS * 3).shape
 
             full_pose = body_model.dict2tensor(poses)
-            start = (min(start, num_frame - 1) + num_frame) % num_frame
-            end = (min(end, num_frame - 1) + num_frame) % num_frame
-            full_pose = full_pose[start:end + 1]
+            start = (min(start, num_frames - 1) + num_frames) % num_frames
```

Comment:
Similar as above, we may leave the handling of `start` and `end` to the slicing function of torch/numpy. Torch/numpy will handle various case for us.

Reply:
I think it is still common that I have a folder of images more than the number of frames I want to visualize. Especially of the images I captured and saved.

Reply:
I think `start` and `end` are fixed in ffmpeg_utils?

Reply:
This will be changed and the users are strongly recommend to use `frame_list` or `image_array` if the existing image number in the folder is more than your `end-start`